### ISSUE TITLE
backrest can't use vss-snapshot with startup

### ISSUE THREAD

**Issue by swartjie** [garethgeorge/backrest#412]
Note: if you have a question or need support please post in the [discussions area](https://github.com/garethgeorge/backrest/discussions).

**Describe the bug**
When I installed the latest version of backrest, It seems as though the VSS Snapshot feature can't be implemented, after I restart the pc. The backup fails if I try and implement it complaining about "VSS error: The caller does not have sufficient backup privileges or is not an administrator: E_ACCESSDENIED"

How can we fix it? Or what can I do?

**To Reproduce**

Install Backrest on windows
Setup repo
Setup backup plan (enable vss snapshot flag)
Reboot
Run Backup

**Expected behavior**
it's supposed to backup the system after making a snapshot.

**Platform Info**
Windows 11

**Comment by daniel-rikowski** [comment]
Do you run backrest as Administrator? The VSS feature requires elevated rights... 
In principle, adding the user to the *Backup Operators* group would suffice, but if you are using a scheduled task, perhaps it's easier to just tick the "Run with highest privileges" checkbox.

**Comment by daniel-l** [comment]
> No, you are right, it's not possible without jumping through some hoops.
> 
> The most straight forward way is to create a scheduled task and set the trigger to "At logon"
> 
> See https://superuser.com/a/929227/2603 (Also consider the comments)
> 
> Also tick the "Run with highest privileges" checkbox.

This is right and the way to got. I have set up two machines this way and it is working flawlessly since weeks.
